measure_all

```diff
@@ -0,0 +1 @@
+docker run -i juliareach
```

Comment:
Does this work, in the sense that  it copies to the calling  machine the proper file? Let me play a little bit on my local machine...

Reply:
in this branch if i now do
 
```julia
$ julia startup.jl
```
it copies the file to the new folder. but this doesn't pass through docker. i haven't tried to run it through docker.

Reply:
hmm perhaps we should add a check if the directory exists.... because if it's ran for the second time,

```julia
$ julia startup.jl
 Activating environment at `~/Projects/ARCH2020_NLN_RE/Project.toml`
ERROR: LoadError: IOError: mkdir: file already exists (EEXIST)

Stacktrace:
 [1] uv_error at ./libuv.jl:97 [inlined]
 [2] mkdir(::String; mode::UInt16) at ./file.jl:177
 [3] mkdir at ./file.jl:170 [inlined]
 [4] main() at /home/mforets/Projects/ARCH2020_NLN_RE/startup.jl:10
 [5] top-level scope at /home/mforets/Projects/ARCH2020_NLN_RE/startup.jl:45
 [6] include(::Function, ::Module, ::String) at ./Base.jl:380
 [7] include(::Module, ::String) at ./Base.jl:368
 [8] exec_options(::Base.JLOptions) at ./client.jl:296
 [9] _start() at ./client.jl:506
in expression starting at /home/mforets/Projects/ARCH2020_NLN_RE/startup.jl:45
```

Reply:
> in this branch if i now do
> 
> ```julia
> $ julia startup.jl
> ```
> 
> it copies the file to the new folder. but this doesn't pass through docker. i haven't tried to run it through docker.

I  agree, you are running locally and not using the docker container.

Following what we currently have in the README.md, I think we require some automatic way of copying back from the container to the docker: `docker cp FANCY_NAME:/result/ ./`.

Incidentally, it would be a good idea to provide details in the `README.md` how to use the script.

Reply:
Currently, I am testing (the docker) using `bash measure_all`...

Reply:
Just pushed some update for `measure_all`. As far as I've checked, it runs the benchmarks, produces the plots, and copies them into the local machine. It's quite pedestrian, but seems to work.